dcos/dcos-ui
# [spike] DCOS-48015: [1.13] Adding Package Repository with priority 0, does not push all packages 1 priority down

Fix repository priority when adding a new repository.

Closes https://jira.mesosphere.com/browse/DCOS-48015

Thanks to @nLight for helping me realize that priority is called index in the API.

Once this is approved, I will create the backports.

## Testing
1. Go to Settings > Package Repositories
2. Add a new Package Repository with priority 0, for example using the name "Universe2" and the uri "https://downloads.mesosphere.com/universe/repo/repo-up-to-1.13.json.gz"
3. Verify that the new package was added at the top

## Trade-offs
None.

## Dependencies
None.

## Screenshots

### Before
![peek 2019-02-01 13-54](https://user-images.githubusercontent.com/40791275/52121709-3b2f9700-2629-11e9-8e4d-bb70592b8c5f.gif)

### After
![peek 2019-02-01 13-55](https://user-images.githubusercontent.com/40791275/52121712-3e2a8780-2629-11e9-97b6-f7013a8c726f.gif)
